CHANGELOG.md

```diff
@@ -4,6 +4,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### 5.8.0
+### Added
+* Service: Add correct routes and auth to agent C# functions. [#2109](https://github.com/microsoft/onefuzz/pull/2109)
+* Service: Port `agent_registration` to C#. [#2107](https://github.com/microsoft/onefuzz/pull/2107)
+* Agent: Add the `dotnet_coverage` task. [#2062](https://github.com/microsoft/onefuzz/pull/2062)
+* Agent: Add multiple ways to specify LibFuzzerDotnetLoader targets. [#2136](https://github.com/microsoft/onefuzz/pull/2136)
+* Agent: Add logging to LibFuzzerDotnetLoader. [#2141](https://github.com/microsoft/onefuzz/pull/2141)
+* Documentation: Added documentation for LibFuzzerDotnetLoader. [#2142](https://github.com/microsoft/onefuzz/pull/2142)
+
+### Changed
+* Service: Add caching to C# storage implementation so repeated queries do not get throttled. [#2102](https://github.com/microsoft/onefuzz/pull/2102)
+* Service: Remove unused poolname validation. [#2094](https://github.com/microsoft/onefuzz/pull/2094)
+* Service: Make the hostbuilder async in C#. [#2122](https://github.com/microsoft/onefuzz/pull/2122)
+* Service: Include State filter when searching for expired tasks and jobs. [#2138](https://github.com/microsoft/onefuzz/pull/2138)
```

Comment:
The state filter change  should be in the fixed section

Reply:
Updated